python/pyarrow/parquet.py

```diff
@@ -124,11 +124,14 @@ class ParquetFile(object):
     memory_map : boolean, default True
         If the source is a file path, use a memory map to read file, which can
         improve performance in some environments
+    enable_buffered_stream : int, default 0
+        The size of the temporary read buffer.
```

Comment:
This comment isn't clear about what the argument is for.

"If positive, perform read buffering when deserializing individual column chunks. Otherwise IO calls are unbuffered"

Other kinds of IO when reading the file won't be affected by this